docs/src/main/paradox/cluster-sharding.md

```diff
@@ -0,0 +1,92 @@
+---
+project.description: Alpakka Kafka provides a module to use Kafka with Akka Cluster External Sharding.
+---
+# Akka Cluster Sharding
+
+Akka Cluster allows the user to use an external shard allocation strategy in order to give the user more control over
```

Comment:
Link to https://doc.akka.io/docs/akka/current/typed/cluster-sharding.html#external-shard-allocation